src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -683,7 +683,7 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
         $dumper = new PhpDumper($container);
 
         if (class_exists('ProxyManager\Configuration')) {
-            $dumper->setProxyDumper(new ProxyDumper());
+            $dumper->setProxyDumper(new ProxyDumper(md5((string) $cache)));
```

Comment:
Is this unique enough? I know for caching keys Symfony uses sha256 but if this is dumped to a filename it should use sha1.